GVRf/Framework/src/org/gearvrf/GVRRenderData.java

```diff
@@ -261,7 +279,12 @@ public boolean isDone() {
             return mFutureMesh.isDone();
         }
     }
-
+    
+    public void addPass(GVRMaterial material, int cullFace) {
+    	NativeRenderData.addPass(getNative(), material.getNative(), cullFace);
+    }
+    
+    
     /**
```

Comment:
True. I commented this as Future Plans when I created the pull request. I planned to make this on next pull request. I implemented it this way because I won't be able to work on this for the next week and I wanted to push it as it is stable.

More rendering states that make sense to set per pass, like polygon offset, are going to be added on RenderPass too.